pootle/apps/pootle_store/views.py

```diff
@@ -629,6 +630,28 @@ def handle_suggestion_comment(request, suggestion, comment):
     if comment_form.is_valid():
         comment_form.save()
 
+        if (action not in ("accepted", "rejected") or
+            not settings.POOTLE_EMAIL_FEEDBACK_ENABLED):
+
+            return
+
+        ctx = {
+            'suggestion_id': suggestion.id,
+            'unit_url': request.build_absolute_uri(unit.get_translate_url()),
+            'comment': comment,
+        }
+        if action == "rejected":
+            message = loader.render_to_string(
+                'editor/email/suggestion_rejected_with_comment.txt', ctx)
+            subject = _(u"Suggestion %s rejected with comment" % suggestion.id)
+        else:
```

Comment:
The aim of that was to keep email threads separate, but since you are pestering about it, it is just quicker to please you all. Removing then.